msu/hooks/skills/skill_container.nut

```diff
@@ -419,9 +418,9 @@
 		]);
 	}
 
-	o.onBeforeTargetHit = function( _caller, _targetEntity, _hitInfo )
+	q.onBeforeTargetHit = @(__original) function( _caller, _targetEntity, _hitInfo )
 	{
-		if (_caller.isAttack())
+		if (_caller.isAttack() && _caller.getDamageType() != null)
```

Comment:
Seems like a bugfix and shouldn't be in this PR

Reply:
As prev